apps/andi/config/integration.exs

```diff
@@ -67,3 +76,20 @@ config :andi, AndiWeb.Endpoint,
   live_view: [
     signing_salt: "SUPER VERY TOP SECRET!!!"
   ]
+
+defmodule Andi.DivoPostgres do
```

Comment:
we can replace this with https://hexdocs.pm/divo_postgres/readme.html#content before we're wrapped up